### ISSUE TITLE
Improve community interactions

### ISSUE THREAD

**Issue by TheJaredWilcurt** [jnordberg/wintersmith#317]
I recently raised an issue to have a simple typo fixed and was met with hostility.

* https://github.com/jnordberg/wintersmith/issues/316

## TLDR:

* **Me:** You put Sass in all caps, it shouldn't be.
* **jnordberg:** Snarky comment explaining: *I don't understand the problem*

(Issue was then closed)

* **Me:** Detailed explanation of the problem and why it should be fixed.
* **jnordberg:** I expect people logging bugs for my project to also fix those bugs, for free.
* **Me:** I refuse to do any more free work for you. If you don't care enough about your own project to fix this, that probably means there isn't much care or attention to detail placed else where in it, making it not worth using.
* **jnordberg:** YOU SUCK! Don't use my poorly maintained project!

(Comments on the issue were then immediately locked).

Way ahead of ya.

**Comment by jnordberg** [comment]
Hey man, I'm sorry I hurt your feelings. If you want the proper capitalisation of your word please create a new issue with and link to where it is so I know where to look, and I'll get to It when I get to it. Or send a patch...
